src/faction_camp.cpp

```diff
@@ -190,458 +192,477 @@ void talk_function::camp_missions( mission_data &mission_key, npc &p )
             entry = entry + _( "\n \nDo you wish to bring your allies back into your party?" );
-            mission_key.text["Recover Ally from Gathering"] = entry;
-            mission_key.push( "Recover Ally from Gathering", _( "Recover Ally from Gathering" ), "", true,
-                              avail );
-        }
-
-        mission_key.text["Distribute Food"] = string_format( _( "Notes:\n"
-                                              "Distribute food to your follower and fill you larders.  Place the food you wish to distribute opposite the tent door between "
-                                              "the manager and wall.\n \n"
-                                              "Effects:\n"
-                                              "> Increases your faction's food supply value which in turn is used to pay laborers for their time\n \n"
-                                              "Must have enjoyability >= -6\n"
-                                              "Perishable food liquidated at penalty depending on upgrades and rot time:\n"
-                                              "> Rotten: 0%%\n"
-                                              "> Rots in < 2 days: 60%%\n"
-                                              "> Rots in < 5 days: 80%%\n \n"
-                                              "Total faction food stock: %d kcal or %d day's rations" ), 10 * camp_food_supply(),
-                                              camp_food_supply( 0, true ) );
-        mission_key.push( "Distribute Food", _( "Distribute Food" ) );
-
-        mission_key.text["Reset Sort Points"] = string_format( _( "Notes:\n"
-                                                "Reset the points that items are sorted to using the [ Menial Labor ] mission.\n \n"
-                                                "Effects:\n"
-                                                "> Assignable Points: food, food for distribution, seeds, weapons, clothing, bionics, "
-                                                "all kinds of tools, wood, trash, books, medication, and ammo.\n"
-                                                "> Items sitting on any type of furniture will not be moved.\n"
-                                                "> Items that are not listed in one of the categories are defaulted to the tools group."
-                                                                ) );
-        mission_key.push( "Reset Sort Points", _( "Reset Sort Points" ), "", false );
+            mission_key.add_return( "Recover Ally from Gathering",
+                                    _( "Recover Ally from Gathering" ), "", entry, avail );
+        }
+
+        entry = string_format( _( "Notes:\n"
+                                  "Distribute food to your follower and fill you larders.  Place "
+                                  "the food you wish to distribute opposite the tent door between "
+                                  "the manager and wall.\n \n"
```

Comment:
Perhaps this text should mention that this is the default food distribution point, but might be changed by sort points.

Reply:
Fixed.